### PR TITLE
fixed regex for rpc definition to allow use of .

### PR DESCRIPTION
**Pull request by jvsteiner**
This is legal syntax which isn't highlighted correctly.  It occurs when using someone elses protobuf definitions, for example from a foreign.api package:

```
rpc MyServiceRoute(MyRequest) returns (foreign.api.Response) { // <- here
	option (google.api.http) = {
		post: "/v1/echo" body: "*"
	};
```

### CODE REVIEW

**Comment by vihangm** [comment]
Can you update the `.YAML-tmLanguage` version of the file and then autogenerate the `.tmLanguage` with https://github.com/SublimeText/PackageDev

Otherwise these changes will get overwritten when someone else autogenerates the `.tmLaguage` from the YAML file.

File: Protobuf.tmLanguage [hunk 1]
```diff
@@ -353,6 +353,24 @@
...
+				</dict>
+				<key>2</key>
+				<dict>
+					<key>name</key>
+					<string>entity.name.tag</string>
+				</dict>
+			</dict>
+			<key>match</key>
+			<string>(syntax)(?=(\s+=\s+)(["]?[A-Za-z][A-Za-z0-9-_.\/]*["]?)(;))</string>
+		</dict>
```
Comment (vihangm) [inline Protobuf.tmLanguage:373]: Did you mean to include these changes? This seems to be missing in the YAML.

**Review by vihangm — CHANGES_REQUESTED** [review]

**Comment by jvsteiner** [comment]
I did not, those were merged later on my local copy

Vihang Mehta wrote:

**Comment by vihangm** [comment]
Abandoning in favor of #13

**Pull request closed without merging**